plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -166,12 +164,18 @@ const blocks = {
 	},
 };
 
+// Intentional separation of cart and checkout entry points to allow for better code splitting.
+const cartAndCheckoutBlocks = {
+	cart: {},
+	checkout: {},
+};
```

Comment:
Have you evaluated if it would be worth including the Mini-Cart Contents block in this bundle? It shares many components and dependencies with the Cart block.

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/48010/commits/c30edd911149c1b27692e0b5a2207210f362238e thanks for calling this out.